lib/r10k/action/deploy/environment.rb

```diff
@@ -86,6 +93,16 @@ def visit_module(mod)
           logger.info "Deploying module #{mod.path}"
           mod.sync
         end
+
+        def write_environment_info!(environment)
+          File.open("#{environment.path}/.r10k-deploy.json", 'w') do |f|
+            deploy_info = environment.info.merge({
+              :deployed_at => Time.new,
```

Comment:
`:deployed_at` indicates that this was the start time (at least to me) but this would run at the end of the environment deploy. In addition I have these wild fantasies of including timing information (for #67). This a bit bike-sheddy, but could we use a name like `:finished_at` or something like that?